infrastructure/src/main/java/org/corfudb/infrastructure/LogReplicationServer.java

```diff
@@ -45,6 +46,10 @@ public class LogReplicationServer extends AbstractServer {
     // node id should be the only identifier for a node in the topology
     private String localNodeId;
 
+    /*
+     * Size bounding LRs client RPC queue, set to be at least that of the sender buffer window.
+     */
+    private static final int MAX_EXECUTOR_QUEUE_SIZE = 5;
```

Comment:
Can we leave out changing this from 10 to 5? Trying to contain the changes for now.